bolt-modules/boltlib/lib/puppet/functions/get_targets.rb

```diff
@@ -3,9 +3,9 @@
 require 'bolt/error'
 
 # Parses common ways of referring to targets and returns an array of Targets.
-# `get_targets('all')` returns an empty array.
 #
-# **NOTE:** Not available in apply block when `future` is true
+# **NOTE:** Calling `get_target('all')` returns an empty array.
```

Comment:
This should be `get_targets` but I don't think it's actually true??

Also we should leave the notes about "Not available in apply blocks" and just remove the "when `future` is true" bit, for each of these functions.